HW02/Dzmitry_Neverov/task2/logs_validator.rb

```diff
@@ -0,0 +1,27 @@
+class LogsValidator
+  attr_reader :logs_to_check
+
+  def initialize(logs_to_check)
+    @logs_to_check = logs_to_check
+  end
+
+  LOG_VALIDATE = %r{
+(?<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}).{5}\[
+(?<date>\d{1,2}/\w{3}/\d{4}\:\d{,2}:\d{,2}:\d{,2}\s\+\d{4})\]\s"POST\s
+(?<url>/\w+/\d+/\w+)\s
+}x.freeze
+
+  def task_2
+    valid_logs = 0
+    logs_to_check.map do |line|
+      next unless line.match?(LOG_VALIDATE)
```

Comment:
Try to implement the following interface 
```suggestion
  def task_2
    valid_logs.map { .. }
  end
  
   private 
   def valid_logs
     logs_to_check.select { ... }
   end
```